content/en/continuous-deployment/plugin-guide/policy-engine/use/_index.md

```diff
@@ -225,13 +223,13 @@ allow {
 
 ### Return a custom message
 
-> Providing a custom message for the `spinnaker.http.authz` package requires Armory Enterprise 2.26.0 or later.
+> Providing a custom message for the `spinnaker.http.authz` package requires Armory Continuous Deployment 2.26.0 or later.
 
 The Policy Engine allows you to return a custom message when an action violates a policy and is blocked. For most packages that you want to enforce policies on, this is done by specifying the message as part of the `deny` block. You can see this in the examples for other policy types on this page.
 
 Returning a custom message works slightly differently for the `spinnaker.http.authz` package because it is based on the `allow` rules rather than `deny` rules.
 
-If a user attempts to perform an action that they are not allowed to, a window appears and displays the custom message you specify. If you are running an Armory Enterprise version earlier than 2.26.0, you cannot specify a custom message. Instead, a generic server error message appears.
+If a user attempts to perform an action that they are not allowed to, a window appears and displays the custom message you specify. If you are running an Armory Continuous Deployment version earlier than 2.26.0, you cannot specify a custom message. Instead, a generic server error message appears.
```

Comment:
```suggestion
If a user attempts to perform an action that they are not allowed to, a window appears and displays the custom message you specify.
```

Versions earlier than 2.26 are out of support at this point so don't see the need to keep this message in this version of the docs (as it'll remain in the previous versions of the docs)

Reply:
Ok. I think Discovery uses 2.25... and yes, they should be looking at the 2.25 docs.